sympy/ntheory/__init__.py

```diff
@@ -25,6 +25,7 @@
     continued_fraction_convergents, continued_fraction
 from .digits import count_digits, digits, is_palindromic
 from .egyptian_fraction import egyptian_fraction
+from .ecm import ecm, Point
```

Comment:
`Point` is a fairly special object: no `y` coordinate, includes `a_24` that is probably meaningless outside of this module. I would prefer not making `Point` public.

Reply:
I was also thinking this. One mere thing shall i remove the examples and test cases of Point class entirely.